src/browser/services/canny.ts

```diff
@@ -74,3 +77,34 @@ export const CannySDK = {
       }
     })
 }
+
+export class CannyLoader extends Component {
+  componentDidMount(): void {
+    CannySDK.init()
+      .then(() => {
+        window.CannyIsLoaded = true
+      })
+      .catch(() => {
+        window.CannyIsLoaded = false
+      })
+  }
+
+  shouldComponentUpdate(): boolean {
+    return false
+  }
+
+  componentWillUnmount(): void {
+    if (canUseDOM()) {
+      delete (window as any).CannyIsLoaded
+      delete (window as any).Canny
+      const cannyTag = document.getElementById('canny-jssdk')
+      if (cannyTag?.outerHTML) {
```

Comment:
hmm, a little unconventional to remove a dom element this way imo, it's more normal to just do something like:

```
elem.parentNode.removeChild(elem)
```

Also, the Intercom & Segment components that work similar to this one don't seem to be removing the script element, so it might be good to make that consistent...

Reply:
I'll leave the script tag in.